interchain/scripts/src/bin/full_deploy.rs

```diff
@@ -23,15 +23,15 @@ fn full_deploy(mut networks: Vec<ChainInfoOwned>) -> anyhow::Result<()> {
         networks = SUPPORTED_CHAINS.iter().map(|x| x.clone().into()).collect();
     }
 
-    let deployment_status = read_deployment()?;
-    if deployment_status.success {
-        log::info!("Do you want to re-deploy to {:?}?", networks);
-        let mut input = String::new();
-        std::io::stdin().read_line(&mut input)?;
-        if input.to_lowercase().contains('n') {
-            return Ok(());
-        }
-    }
+    // let deployment_status = read_deployment()?;
```

Comment:
what's happening here ? Is this part not relevant anymore ?

Reply:
I don't have deployment status file and it's not inside this repo either. This script was erroring without this file, so I commented it out for now.